.gitignore

```diff
@@ -19,5 +19,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-yarn.lock
```

Comment:
yay!

Reply:
Gonna add package-lock.json juuuust in case.